.github/issue_and_pr_commands.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "type": "label",
+    "name": "datasource/Redshift",
+    "action": "addToProject",
+    "addToProject": {
+      "url": "https://github.com/orgs/grafana/projects/97"
+    }
+  },
+    {
```

Comment:
Also adding triggers for `type/docs` label. We add this command to all repositories.